core/src/main/scala/org/apache/spark/Logging.scala

```diff
@@ -36,7 +36,7 @@ import org.apache.spark.util.Utils
 trait Logging {
   // Make the log field transient so that objects with Logging can
   // be serialized and used on another machine
-  @transient private var log_ : Logger = null
+  @transient private var _log: Logger = null
```

Comment:
It is less big of a deal there since it is already done and it is small. But feel free to do whatever you deem fit.